### PR TITLE
feynhiggs: add new package

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/feynhiggs/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        of the masses, mixings and much more of the Higgs bosons in the MSSM
+        with real/complex parameters at the highest level of accuracy."""
+
+    homepage = "https://wwwth.mpp.mpg.de/members/heinemey/feynhiggs/cFeynHiggs.html"
+    url      = "https://wwwth.mpp.mpg.de/members/heinemey/feynhiggs/newversion/FeynHiggs-2.18.1.tar.gz"
+
+    maintainers = ['vvolkl']
+
+    version('2.18.1', sha256='a9cdc4e2759f96fb9bd981b7be1ba8df070fb20c46d5b95e0c9700fccafe5ee6')
+
```
Comment (vvolkl) [inline var/spack/repos/builtin/packages/feynhiggs/package.py:21]:
```suggestion
```

File: var/spack/repos/builtin/packages/feynhiggs/package.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    def configure_args(self):
+        args = ['FFLAGS=-fPIC CFLAGS=-fPIC']
+        return args
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/feynhiggs/package.py]: Normally each argument is added to the list as a separate entry.  Also, if you want to have the args list on a single line, you could just do something along the lines `return ['FFLAGS=-fPIC', 'CFLAGS=-fPIC']`.

Comment (vvolkl) [reply var/spack/repos/builtin/packages/feynhiggs/package.py]:
```suggestion
        return ['FFLAGS=-fPIC', 'CFLAGS=-fPIC']
```

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**